### ISSUE TITLE
false positive: unused import when using a class

### ISSUE THREAD

**Issue by tonyraoul** [vuejs/eslint-plugin-vue#472]
Code
```
import axios from 'axios'
import MockAdapter from 'axios-mock-adapter'

let mock = new MockAdapter(axios)
```
it doesn't consider a class instance as usage of import

Output
```
  ✘  http://eslint.org/docs/rules/no-unused-vars  'MockAdapter' is defined but never
 used
```
